src/bundle/Resources/config/routing.yml

```
@@ -398,6 +398,15 @@ ibexa.rest.languages.view:
 # Locations
 
 
+ibexa.rest.move_location:
+    path: /content/locations/{locationPath}
+    defaults:
+        _controller: Ibexa\Rest\Server\Controller\Location:moveLocation
```

Legacy controller syntax used + move the value to `controller` property.

```suggestion
    controller: Ibexa\Rest\Server\Controller\Location::moveLocation
```